src/constants/SupportMessages.ts

```diff
@@ -0,0 +1,6 @@
+export const SupportMessages = {
+  API_KEY_NOT_FOUND: `
+Clerk API key not found during initialization.
```

Comment:
Should we include a helpful message so that one knows what environment variable to set? (`CLERK_API_KEY`)